plugins/woocommerce/tests/php/src/Internal/FraudProtection/CheckoutEventTrackerTest.php

```
@@ -59,118 +54,232 @@ public function setUp(): void {
-			'action'           => 'field_update',
-			'billing_email'    => 'test@example.com',
-			'shipping_methods' => array( 'flat_rate:1' => 'Flat rate' ),
-			'session'          => array( 'session_id' => 'test-session' ),
-		);
+		// Mock scheduler to capture event data.
+		$captured_event_data = null;
+		$this->mock_dispatcher
+			->expects( $this->once() )
+			->method( 'dispatch_event' )
+			->willReturnCallback(
+				function ( $event_type, $event_data ) use ( &$captured_event_data ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$event_type'. (undefined)

(UnusedFormalParameter)